proto/plan.proto

```diff
@@ -246,6 +246,7 @@ message PartitionInfo {
 	repeated PartitionItem partitions = 7;
 	int64 algorithm = 8;
 	bool isSubPartition = 9;
+	string partitionMsg = 10;
```

Comment:
suggest use "_" to split words in  proto file

Reply:
ok.no problem!